test/integration/targets/win_module_utils/library/uses_local_utils.ps1

```diff
@@ -0,0 +1,9 @@
+#! powershell
+
+# use different cases, spacing and plural of 'module' to exercise flexible powershell dialect
+#ReQuiReS -Module Ansible.ModuleUtils.PowerShellLegacy
+#Requires -Module Ansible.ModuleUtils.ValidTestModule
```

Comment:
Yeah, I had just removed the actual different form temporarily and forgot to fix it. Fixed